src/Aks/Aks/Commands/SetAzureRmAks.cs

```diff
@@ -259,11 +266,81 @@ public override void ExecuteCmdlet()
                             }
                         }
 
+                        if (this.IsParameterBound(c => c.KubernetesVersion) && this.IsParameterBound(c => c.NodeImageOnly))
+                        {
+                            throw new AzPSArgumentException(Resources.UpdateKubernetesVersionAndNodeImageOnlyConflict, "KubernetesVersion");
+                        }
+
+                        bool AllAgentPoolVirtualMachineScaleSets = cluster.AgentPoolProfiles.All(c => c.Type.ToLower().Equals("virtualmachinescalesets"));
```

Comment:
lower case local variable
```suggestion
                        bool allAgentPoolVirtualMachineScaleSets = cluster.AgentPoolProfiles.All(c => c.Type.ToLower().Equals("virtualmachinescalesets"));
```